packages/picasso/src/MenuItem/MenuItem.tsx

```diff
@@ -58,6 +59,8 @@ export interface Props extends BaseProps, TextLabelProps, MenuItemAttributes {
   description?: ReactNode
   /** Render an `<Icon />` */
   icon?: ReactElement
+  /** Render an Avatar */
```

Comment:
nit (just a consistency)
```suggestion
  /** Render an <Avatar /> */
```

Reply:
Done :+1: